go/kbfs/simplefs/simplefs.go

```diff
@@ -3700,33 +3698,76 @@ func (k *SimpleFS) SimpleFSGetArchiveStatus(ctx context.Context) (
 				statusJob.SkippedCount++
 			}
 		}
-		{ // get current revision
-			fb, _, err := k.getFolderBranchFromPath(ctx,
-				keybase1.NewPathWithKbfs(keybase1.KBFSPath{
-					Path: stateJob.Desc.KbfsPathWithRevision.Path}))
-			if err != nil {
-				return keybase1.SimpleFSArchiveStatus{}, err
-			}
-			if fb == (data.FolderBranch{}) {
-				return keybase1.SimpleFSArchiveStatus{}, nil
-			}
-			status, _, err := k.config.KBFSOps().FolderStatus(ctx, fb)
-			if err != nil {
-				return keybase1.SimpleFSArchiveStatus{}, err
-			}
-			statusJob.CurrentTLFRevision = keybase1.KBFSRevision(status.Revision)
-		}
 		if errState, ok := errorStates[jobID]; ok {
 			statusJob.Error = &keybase1.SimpleFSArchiveJobErrorState{
 				Error:     errState.err.Error(),
 				NextRetry: keybase1.ToTime(errState.nextRetry),
 			}
 		}
-		status.Jobs[jobID] = statusJob
+		status.Jobs = append(status.Jobs, statusJob)
 	}
+	sort.Slice(status.Jobs, func(i, j int) bool {
+		return status.Jobs[i].Desc.StartTime.Before(status.Jobs[j].Desc.StartTime)
+	})
 	return status, nil
 }
 
+// SimpleFSGetArchiveStatus implements the SimpleFSInterface.
+func (k *SimpleFS) SimpleFSGetArchiveStatus(ctx context.Context) (
+	status keybase1.SimpleFSArchiveStatus, err error) {
+	ctx = k.makeContext(ctx)
+	state, errorStates := k.archiveManager.getCurrentState(ctx)
+	return k.archiveStateToStatus(ctx, state, errorStates)
+}
+
+// SimpleFSGetArchiveJobFreshness implements the SimpleFSInterface.
+func (k *SimpleFS) SimpleFSGetArchiveJobFreshness(ctx context.Context, jobID string) (keybase1.SimpleFSArchiveJobFreshness, error) {
```

Comment:
Is it intentional that the command line implementation is still only calling the old endpoint, and so is no longer getting freshness information like it was before?  It looks like it depends on the `CurrentTLFRevision` field that is no longer being filled in by the other call.

Reply:
Good catch! I forgot; fixed